woocommerce/woocommerce-ios
# Product List: add empty state illustration

Resolves #1686 

## Changes

- Added image assets for image catalog `woo-empty-products` in Dark and Light modes
- Used the new empty Products image in the Products tab empty state overlay

## Testing

Prerequisite: the store has no Products (or hard code to show the empty state)

- Launch the app
- Go to Products tab --> an empty state image should be shown above the "No products yet" message

## Example screenshots

Light | Dark
-- | --
![Simulator Screen Shot - iPhone 8 - 2019-12-31 at 11 16 03](https://user-images.githubusercontent.com/1945542/71608999-01ac3680-2bc0-11ea-8580-25f60cae45e4.png) | ![Simulator Screen Shot - iPhone 8 - 2019-12-31 at 11 16 13](https://user-images.githubusercontent.com/1945542/71609000-01ac3680-2bc0-11ea-9279-d5df30b33de2.png)


Update release notes:
